media-libs/x265/x265-9999.ebuild

```diff
@@ -177,6 +176,8 @@ multilib_src_configure() {
 	elif [[ ${ABI} = arm ]] ; then
 		myabicmakeargs+=( -DENABLE_ASSEMBLY=$(usex pic OFF $(usex cpu_flags_arm_neon ON OFF)) )
 		use cpu_flags_arm_neon && use pic && ewarn "PIC has been requested but arm neon asm is not PIC-safe, disabling it."
+	else
+		myabicmakeargs+=( -DENABLE_TESTS=$(usex test) )
```

Comment:
you are doing the opposite here it seems: if tests require assembly to be on, which is default enabled on x86*/arm* in the cmake files then this will 'else' will never be reached

plus it won't fix all cases since assembly may be disabled further down in the chain too